src/core/api/log.go

```diff
@@ -87,17 +89,21 @@ func (l *LogAPI) Get() {
 			return
 		}
 
-		if len(projects) == 0 {
+		ids := []int64{}
+		for _, project := range projects {
+			roles := l.SecurityCtx.GetProjectRoles(project.ProjectID)
+			if len(roles) > 0 && roles[0] != common.RoleLimitedGuest {
```

Comment:
You should use `HasProjectPermission` from `BaseController` to check the user has logs permission.

Reply:
Hi, mhuangpivotal, can you resolve the problem I replied asap so that this fix can be released with 1.10.1 ?

Reply:
Hi @heww, thanks for the review. I have pushed the requested change.

Reply:
@mhuangpivotal Thank you very much.

The swagger-codegen-cli-2.3.1.jar download failed from the http://central.maven.org which cause the ci failed and today a PR fix this issue merged in release-1.10.0.

Can you rebase this PR with release-1.10.0 branch? Thanks!

Reply:
@heww done